keyboards/6lit/keymaps/default/keymap.c

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2018
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_split( /* Base */
+    KC_F13,   KC_F14,   KC_F15,   KC_F16,   KC_F17,   KC_F18, \
+    KC_F19,   KC_F20,   KC_F21,   KC_F22,   KC_F23,   KC_F24  \
+  ),
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
+	// keyevent_t event = record->event;
+  //
+	// switch (id) {
+  //
+	// }
+	return MACRO_NONE;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  return true;
+}
+
+void matrix_init_user(void) {
+
+}
+
+void matrix_scan_user(void) {
+  // jump to bootloaer when all keys are pressed
+  if (matrix_get_row(0) == 0b111 && matrix_get_row(1) == 0b111) {
```

Comment:
Have you see the new bootmagic lite option? 
https://docs.qmk.fm/#/feature_bootmagic?id=bootmagic-lite

Reply:
I hadn't, and it seems like an awesome feature for larger boards (like some of the other 40percent.club boards). However as this board has such limited amount of keys, and its primary use is as a macro pad, it would seem difficult to map it to the functionality. Its quite nice to just turn it upside down, push, and mash all keys at the same time.

Happy to remove the code from the default keymaps and rely on the user forcing a hardware reset to bootloader if required?

Reply:
Yeah, the lite part is just the "reset to bootloader" bit.  

Also, keep in mind that it only works when plugging in the board.  

And you can customize the key for it, too.

If that's what you want, let me know, and I'll merge it